libr/cons/cons.c

```diff
@@ -696,12 +715,25 @@ R_API void r_cons_flush() {
 		r_cons_reset ();
 		return;
 	}
+ 	if (lastMatters ()) {
+		if (!CTX (lastMode)) {
```

Comment:
just do `if (lastMatters () && !CTX (lastMode))` and set `lastMode = true` inside the inner if.

Reply:
thats not the same

Reply:
refactored the code, now its cleaner, thanks for the hint